### PR TITLE
AWS Basic UPI

### PR DESCRIPTION
**Pull request by yunjiang29**
Adding basic UPI workflow for QE e2e test.

Resolve issue https://issues.redhat.com/browse/OCPQE-6644

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu @gpei PATL.

File: ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh [hunk 1]
```diff
@@ -116,6 +116,10 @@ PRIVATE_SUBNET_1="$(echo "${PRIVATE_SUBNETS}" | sed 's/"//g' | cut -d, -f2)"
 PRIVATE_SUBNET_2="$(echo "${PRIVATE_SUBNETS}" | sed 's/"//g' | cut -d, -f3)"
 PUBLIC_SUBNETS="$(echo "${VPC_JSON}" | jq '.[] | select(.OutputKey == "PublicSubnetIds").OutputValue')"  # explicitly keeping wrapping quotes
 
+# Adapt step aws-provision-tags-for-byo-vpc, which is required by Ingress operator testing.
```
Comment (gpei) [inline ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh:119]: @yunjiang29 could you help reminding me the background of this?
Adding tags to the `VPC` and `private subnets` is required for a normal UPI AWS install or just for our E2E testing?

Comment (yunjiang29) [reply ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh:119]: @gpei for out e2e test only, see https://github.com/openshift/release/pull/32597

Comment (gpei) [reply ci-operator/step-registry/upi/install/aws/cluster/upi-install-aws-cluster-commands.sh:119]: got it, thanks

**Pull request merged**